### PR TITLE
Feature/rewrite chat UI to jsx

### PR DESCRIPTION
**Pull request by nddmanh**
Chat UI rewrite:
image

### CODE REVIEW

**Comment by ppppbn** [comment]
Too many commits. Some are not belong to this PR

**Comment by ppppbn** [comment]
Last commit message is non meaningful

File: src/modules/playRoom/components/ChatBox.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```
Comment (ppppbn) [inline src/modules/playRoom/components/ChatBox.js:1]: you don't need to import React here as you use functional components

File: src/modules/playRoom/components/ChatBox.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+const ChatBox = () => {
+  return (
+    <div className="noti">
+      <div className="chat-box">
+
```
Comment (ppppbn) [inline src/modules/playRoom/components/ChatBox.js:7]: remove unnecessary spaces & lines

File: src/modules/playRoom/components/ChatBox.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        
+        <div className="client">
+          <div className="client-info">
+            <h2>THÔNG BÁO</h2>
+          </div>
+        </div>
+        
+        <div className="scroll">
+          <div className="chats">
+
```
Comment (ppppbn) [inline src/modules/playRoom/components/ChatBox.js:16]: here, remove it

File: src/modules/playRoom/components/ChatBox.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+          </div>
+        </div>
+        
+        <form className="chat-input">
+          <input type="text" placeholder="Enter Message" />
+          <button type="submit" className="send-btn">
+            <img src="./send.png" alt="send-btn" />
+          </button>
+        </form>
+
```
Comment (ppppbn) [inline src/modules/playRoom/components/ChatBox.js:43]: this too

File: src/modules/playRoom/components/PickItem.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
```
Comment (ppppbn) [inline src/modules/playRoom/components/PickItem.js:1]: remove React

File: src/modules/playRoom/components/PickItem.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+  const handleOk = () => {
+    setIsModalVisible(false);
+  };
+
+  const handleCancel = () => {
+    setIsModalVisible(false);
+  };
+  
+
```
Comment (ppppbn) [inline src/modules/playRoom/components/PickItem.js:19]: remove unnecessary line

File: src/modules/playRoom/components/PickItem.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import { Modal, Input } from 'antd';
```
Comment (ppppbn) [inline src/modules/playRoom/components/PickItem.js:2]: Starting from here, the files belong to another PR. Please refactor your PRs

File: src/modules/playRoom/components/PickItem.js [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+  const handleCancel = () => {
+    setIsModalVisible(false);
+  };
+  
+
+  return (
+    <>
+      <div class="select"  onClick={showModal} >                
+        <input type="image" className="img-select" src={img.src} alt="" name="" value={img.value} />
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickItem.js:23]: Do not leave attributes ```alt```, ```name``` empty, you can remove the attributes if not necessary

File: src/modules/playRoom/components/PickList.js [hunk 9]
```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickList.js:1]: Remove ```React```

File: src/modules/playRoom/components/PickList.js [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+import { Row, Col } from 'antd';
+import PickItem from './PickItem';
+
+const PickList = () => {
+  const [imagesPick, setImagesPick] = useState([
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickList.js:6]: You can define default ```imagesPick``` value in a constant file

File: src/modules/playRoom/pages/PlayRoom.css [hunk 11]
```diff
@@ -0,0 +1,175 @@
+
+#game {
```
Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.css:2]: Use class instead of id in css, and we use ```scss``` in this project

File: src/modules/playRoom/pages/PlayRoom.js [hunk 12]
```diff
@@ -1,8 +1,22 @@
 import React, { useState, useEffect } from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.js:1]: Remove ```React```, ```useState```, ```useEffect```

File: src/modules/playRoom/pages/PlayRoom.js [hunk 13]
```diff
@@ -1,8 +1,22 @@
...
+import PickList from '../components/PickList';
+import './PlayRoom.css';
+import ChatBox from '../components/ChatBox';
 
-function PlayRoom(props) {
+function PlayRoom() {
   return (
-    <div></div>
+    <Row justify="center" gutter={[32, 32]} >
+      <Col xs={24} sm={24} md={24} xl={6} >
```
Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.js:10]: Why we have empty ```Col``` here ?
